Makefile

```
@@ -1024,11 +1025,15 @@ publish-ci: image-ci
 # Docker image build for Teleport Operator
 .PHONY: image-operator-ci
 image-operator-ci:
-	make -C operator docker-build IMG=$(DOCKER_IMAGE_OPERATOR_CI):$(VERSION)
+	make -C operator docker-build IMG=$(DOCKER_IMAGE_OPERATOR_STAGING):$(VERSION)
 
 .PHONY: publish-operator-ci
 publish-operator-ci: image-operator-ci
-	docker push $(DOCKER_IMAGE_OPERATOR_CI):$(VERSION)
+	@if DOCKER_CLI_EXPERIMENTAL=enabled docker manifest inspect $(DOCKER_IMAGE_OPERATOR_STAGING):$(VERSION) 2>&1 >/dev/null; then\
```

```suggestion
	@if DOCKER_CLI_EXPERIMENTAL=enabled docker manifest inspect "$(DOCKER_IMAGE_OPERATOR_STAGING):$(VERSION)" >/dev/null 2>&1; then \
```

If you meant to discard all the output, the order matters. Let's add some quoting for good measure as well.

https://stackoverflow.com/a/25259518

---

Thanks! Updated in https://github.com/gravitational/teleport/pull/15275/commits/7ee8b32676858a7324edc4f376525c8e7327ee56